hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaDistributionSet.java

```diff
@@ -347,14 +347,14 @@ public void fireUpdateEvent(final DescriptorEvent descriptorEvent) {
                 new DistributionSetUpdateEvent(this, EventPublisherHolder.getInstance().getApplicationId()));
 
         if (isSoftDeleted(descriptorEvent)) {
-            publishEventWithEventPublisher(new DistributionSetDeletedEvent(getTenant(), getId(),
+            publishEventWithEventPublisher(new DistributionSetDeletedEvent(getTenant(), getId(), getClass().getName(),
```

Comment:
`getClass()` is returning the runtime type. Is this intendet? The other changed classes are returning a fixed type e.g. `A.class.getClass()`